CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
+- A new parameter `run_mt_for_wes` to turn on mitochondrial analysis for WES cases [#552](https://github.com/nf-core/raredisease/pull/552)
```

Comment:
I assume that it would work for other targeted approaches as well?  We could tailor it to say target instead of wes

Reply:
Yupe, good point!